atom/browser/api/atom_api_browser_window_mac.mm

```diff
@@ -103,7 +101,9 @@ - (NSView*)hitTest:(NSPoint)aPoint {
 
   // Draggable regions is implemented by having the whole web view draggable
   // (mouseDownCanMoveWindow) and overlaying regions that are not draggable.
-  draggable_regions_ = regions;
+  draggable_regions_.clear();
+  for (const auto& r : regions)
+    draggable_regions_.push_back(r.Clone());
```

Comment:
could use `swap`, to avoid `clear` and `push_back`

Reply:
we have to `Clone` every element though... i'm not sure I understand how `swap` would accomplish that?

Reply:
Ah vector of mojo ptrs, nvm.